src/styles/foundation/_focus-ring.scss

```diff
@@ -0,0 +1,25 @@
+/// Sets the focus ring for an interactive element
+/// @param {String} $size - The size of the border radius on the focus ring.
+///
+
+@mixin focus-ring($size: 'base') {
+  position: relative;
+
+  &::after {
+    content: var(--p-focus-ring-content, none);
+    position: absolute;
+    top: rem(-3px);
+    right: rem(-3px);
+    bottom: rem(-3px);
+    left: rem(-3px);
+    display: block;
+    border: rem(2px) solid var(--p-interactive-focus);
+    @if $size == 'wide' {
+      border-radius: calc(var(--p-border-radius-wide) + 3px);
```

Comment:
Do we want to convert our borders and other thin lines and outer detail to rems?

Reply:
Yeah, we do. I think we'll need to add a JavaScript `rem` function.

Reply:
In this case, since we know the value and this focus ring is only used with theming we could hardcode for now.

Reply:
Never mind, decided to convert in JS